library/bignum_mod_raw.c

```diff
@@ -127,7 +127,40 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+int mbedtls_mpi_mod_raw_to_mont_rep( mbedtls_mpi_uint *X,
+                                     const mbedtls_mpi_mod_modulus *m )
+{
+    const mbedtls_mpi_uint one = 1;
+    const size_t t_sz = ( m->limbs * 2 ) + 1;
+    mbedtls_mpi_uint *t;
+
+    if( ( t = (mbedtls_mpi_uint*)mbedtls_calloc( t_sz, ciL ) ) == NULL )
```

Comment:
could fix (i.e. only if making other changes):
```suggestion
    if( ( t = (mbedtls_mpi_uint *)mbedtls_calloc( t_sz, ciL ) ) == NULL )
```